tests/integ/modin/frame/test_apply.py

```diff
@@ -603,6 +603,7 @@ def g(v):
 ]
 
 
+@pytest.mark.modin_sp_short_regress
 @pytest.mark.parametrize("data, apply_func", TRANSFORM_DATA_FUNC_MAP)
 @sql_count_checker(query_count=0)
```

Comment:
Can you also get one from groupby apply?

Reply:
Now groupby apply and some apply tests are failing. We will add it in the coming PRs if we can fix those tests.

Reply:
Yes, @sfc-gh-lmukhopadhyay mentioned about this to me, she will tickets for the each category of test failure for tracking